pkg/subnet/kube/kube.go

```diff
@@ -160,6 +162,7 @@ func NewSubnetManager(ctx context.Context, apiUrl, kubeconfig, prefix, netConfPa
 func newKubeSubnetManager(ctx context.Context, c clientset.Interface, sc *subnet.Config, nodeName, prefix string, useMultiClusterCidr bool) (*kubeSubnetManager, error) {
 	var err error
 	var ksm kubeSubnetManager
+	ksm.annotationPrefix = prefix
```

Comment:
Why do you need this? `ksm.annotations` is already included the prefix

Reply:
I need to store the prefix that is passed newKubeSubnetManager in order to parse the annotations for the stored MAC address. Otherwise i would need to look at ksm.annotations and split any of them to get the prefix.

Reply:
ok, you have a point. Actually, it is strange that we allow to configure this